### PR TITLE
Brainandbeads

### PR DESCRIPTION
**Pull request by teddyseyed**
Adding in the 2 driver files (under libs --> bead-drivers)

### CODE REVIEW

File: libs/bead-motion/device.d.ts [hunk 1]
```diff
@@ -31,65 +31,61 @@ declare namespace input {
...
-    //% shim=pxt::getButtonByPin(PIN_D7,BUTTON_ACTIVE_LOW_PULL_UP)
-    //% parts="buttons"
-    const touchD7: TouchButton;
+    //% block="touch D7" fixedInstance
+    //% shim=pxt::getTouchButton(PIN_A3)
+    const touchA3: TouchButton;
 
     /**
-     * Capacitive pin D8
+     * Capacitive pin A8
```
Comment (pelikhan) [inline libs/bead-motion/device.d.ts]: A4

File: libs/bead-motion/device.d.ts [hunk 2]
```diff
@@ -31,65 +31,61 @@ declare namespace input {
...
-    //% shim=pxt::getButtonByPin(PIN_D6,BUTTON_ACTIVE_LOW_PULL_UP)
-    //% parts="buttons"
-    const touchD6: TouchButton;
+    //% block="touch D6" fixedInstance
+    //% shim=pxt::getTouchButton(PIN_A2)
+    const touchA2: TouchButton;
 
     /**
-      * Capacitive pin D7
+      * Capacitive pin A7
```
Comment (pelikhan) [inline libs/bead-motion/device.d.ts]: looks like comments need to be updated

File: libs/bead-brain/config.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    export const PIN_D3 = DAL.PA05;
+    export const PIN_D4 = DAL.PB08;
+    export const PIN_D5 = DAL.PB09;
+    export const PIN_D6 = DAL.PA04;
+    export const PIN_D7 = DAL.PB03;
+    export const PIN_D13 = DAL.PA17;
+
+    export const PIN_TX = DAL.PB08;  
+    export const PIN_SCK = DAL.PA21;
+    export const PIN_DISPLAY_CS= DAL.PB22;
```
Comment (pelikhan) [inline libs/bead-brain/config.ts]: do we need this?

Comment (teddyseyed) [reply libs/bead-brain/config.ts]: eh, not really right now...i was keeping it for when a display is hooked up to this eventually

**Comment by pelikhan** [comment]
Remove it.

**Comment by pelikhan** [comment]
https://travis-ci.org/Microsoft/pxt-maker/builds/510150843#L805

File: libs/bead-brain/board.json [hunk 4]
```diff
@@ -0,0 +1,186 @@
+{
+  "PART_ID": {
```
Comment (pelikhan) [inline libs/bead-brain/board.json]: not sure what this is

**Pull request merged**